repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publications-overview.ftl

```diff
@@ -0,0 +1,56 @@
+<#-- DECLARATIONS: -->
+<#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"] />
+<#assign fmt=JspTaglibs ["http://java.sun.com/jsp/jstl/fmt"] />
+<#assign truncate="uk.nhs.digital.ps.directives.TruncateFormatterDirective"?new() >
+
+<#macro restrictableDate date>
+    <#assign formatRestrictableDate="uk.nhs.digital.ps.directives.RestrictableDateFormatterDirective"?new() />
+    <#if date??>
+        <@formatRestrictableDate value=date/>
+    </#if>
+</#macro>
+
+
+<#-- @ftlvariable name="publication" type="uk.nhs.digital.ps.beans.Publication" -->
```

Comment:
remove unused code

Reply:
Actually, this only looks like a commented out code but is, in fact, a declaration that binds a type to a variable name so that IDEs can perform auto-completion as described in [Hippo documentation](https://www.onehippo.org/library/development/develop-with-intellij.html).

If that's okay with you I'd leave it in as it's [a quite helpful feature](https://infinityworks.slack.com/files/U6Y7307EX/F7XLDGB4M/pasted_image_at_2017_11_09_11_01_am.png).